src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -34,9 +76,9 @@ export class MessagesComponent implements OnInit {
   deleteMessage() {
     // noinspection JSIgnoredPromiseFromCall
     this.viewService.msgIo.deleteMessage(this.messageToDelete).then(() => {
-      // TODO: Add a toast on successful deletion
+      this.toastr.success('Message deleted successfully', 'Poof!');
     }).catch(e => {
```

Comment:
Not sure how I feel about this because of the fact that the toast disappears on its own makes error reporting more difficult.
```suggestion
    }).catch(e => {
      AuthService.displayError(e);
```